services/core/PlatformDriverAgent/platform_driver/interfaces/bacnet.py

```diff
@@ -98,9 +98,8 @@ def ping_target(self):
             pinged = True
         except errors.Unreachable:
             _log.warning("Unable to reach BACnet proxy.")
-
-        except errors.VIPError:
-            _log.warning("Error trying to ping device.")
+        except (Exception, gevent.Timeout) as e:
+            _log.warning(f"Error trying to ping device: {e}")
```

Comment:
e won't have the device address or the device_id.  Please change the warning to be for the device.